CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### PaymentSheet
+* [CHANGED][11112](https://github.com/stripe/stripe-android/pull/11112) Restricted negative initialization amounts.
```

Comment:
Can you be more specific here? Maybe "Restrict negative initialization amounts during configuration of `PaymentSheet`, `PaymentSheet.FlowController`, and `EmbeddedPaymentElement` when using `PaymentSheet.IntentConfiguration`."